code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -438,51 +436,28 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 //ZIPPO//
 /////////
 /obj/item/weapon/lighter
-	name = "cheap lighter"
-	desc = "A cheap-as-free lighter."
-	icon = 'icons/obj/items.dmi'
-	icon_state = "lighter-g"
-	item_state = "lighter-g"
-	var/icon_on = "lighter-g-on"
-	var/icon_off = "lighter-g"
+	name = "\improper Zippo lighter"
+	desc = "The zippo."
+	icon = 'icons/obj/cigarettes.dmi'
+	icon_state = "zippo"
+	item_state = "zippo"
 	w_class = 1
-	throwforce = 0
 	flags = CONDUCT
 	slot_flags = SLOT_BELT
-	attack_verb = null
 	var/lit = 0
 
-/obj/item/weapon/lighter/grayscale
+/obj/item/weapon/lighter/greyscale
 	name = "cheap lighter"
-	desc ="A cheap-as-free lighter."
-	icon = 'icons/obj/cigarettes.dmi'
-	icon_state = "lighter-off"
-	item_state = "lighter-off"
-	icon_on = "lighter-on"
-	icon_off = "lighter-off"
+	desc = "A cheap-as-free lighter."
+	icon_state = "lighter"
 
-/obj/item/weapon/lighter/grayscale/New()
+/obj/item/weapon/lighter/greyscale/New()
 	var/icon/overlay = new /icon('icons/obj/cigarettes.dmi',"lighter-overlay")

```

Comment:
In theory, however ColorTone is an /icon proc and [I presume you used it for a reason.](https://github.com/tgstation/-tg-station/pull/8076#issuecomment-76564394)